internal/app/agent/service.go

```diff
@@ -250,8 +250,12 @@ func (a *agent) Config() config.Config {
 	return *a.config
 }
 
-func (a *agent) Services() map[string]*services.Service {
-	return a.servs
+func (a *agent) Services() []*services.Service {
```

Comment:
Do you need a pointer here?

Reply:
I like pointer here, dont like to transfer the hole list